lib/models/Message.js

```diff
@@ -109,6 +109,45 @@ export default class Message extends Model {
   }
 
   /**
+   * Updates the isFlagged field of this {@link Message}.
+   *
+   * @param [isFlagged=false] {Boolean} The isFlagged field of the message.
+   *
+   * @return {Promise} A {@link Promise}, as per {@link Client.updateMessage}.
+   *
+   * @see Client#updateMessage
+   */
+  setIsFlagged(isFlagged) {
+    return this._jmap.updateMessage(this.id, { isFlagged: isFlagged || false });
```

Comment:
You actually want a `boolean` here, so you can use your preferred feature of the JS language: the `!!` operator. Your code can become: `return this._jmap.updateMessage(this.id, { isFlagged: !!isFlagged });`

Same comment for the other methods.